libs/output-mapping/src/Configuration/Table/BaseConfiguration.php

```diff
@@ -47,18 +50,114 @@ public static function configureNode(ArrayNodeDefinition $node): void
                     ->end()
                 ->end()
+                ->arrayNode('delete_where')
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('changed_since')->end()
+                            ->scalarNode('changed_until')->end()
+                            ->arrayNode('where_filters')
+                                ->prototype('array')
+                                    ->children()
+                                        ->scalarNode('column')->isRequired()->cannotBeEmpty()->end()
+                                        ->scalarNode('operator')
+                                            ->defaultValue(self::DEFAULT_DELETE_WHERE_OPERATOR)
+                                            ->validate()
+                                                ->ifNotInArray(self::ALLOWED_DELETE_WHERE_OPERATORS)
+                                                ->thenInvalid(
+                                                    'Invalid operator %s. Valid values are "' .
+                                                        implode('" or "', self::ALLOWED_DELETE_WHERE_OPERATORS) . '".',
+                                                )
+                                            ->end()
+                                        ->end()
+                                        ->arrayNode('values_from_set')
+                                            ->prototype('scalar')->end()
+                                        ->end()
+                                        ->arrayNode('values_from_workspace')
+                                            ->children()
+                                                ->scalarNode('id')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('table')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('column')->end()
+                                            ->end()
+                                        ->end()
+                                        ->arrayNode('values_from_storage')
+                                            ->children()
+                                                ->scalarNode('bucket')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('table')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('column')->end()
+                                            ->end()
+                                        ->end()
+                                    ->end()
+                                    ->validate()
+                                        ->always(function ($v) {
+                                            if (empty($v['values_from_set'])) {
+                                                unset($v['values_from_set']);
```

Comment:
jn tohle je na hovno no, ale nevím jak to líp vyřešit :+1:

Reply:
Jako na hovno to je, jinak to asi vyresit nejde.
Navic jsem koukal, ze to stejne tak delame na X mistech https://github.com/search?q=org%3Akeboola%20unset(%24v&type=code